charts/nuts-node/values.yaml

```diff
@@ -99,8 +120,11 @@ nuts:
     # Everything below `nuts.config` will be placed into the `nuts.yaml` config file of the Pods
     datadir: /opt/nuts/data
     http:
+      internal:
+        address: :8081
+      external:
+        address: :8080
       default:
-        address: :1323
         cors:
           origin: "https://chart-example.local"
```

Comment:
the http.default section no longer exists

Reply:
In the files README.rst komt nog de tekst voor:

> As a general safety precaution ``auth.contractvalidators`` ignores the ``dummy`` option if configured,
requesting an access token from another node on ``/n2n/auth/v1/accesstoken`` does not return any error details,
``auth.accesstokenlifespan`` is always 60 seconds, ``http.default.cors.origin`` does not allow a wildcard (``*``),
json-ld context can only be downloaded from trusted domains configured in ``jsonld.contexts.remoteallowlist``,
and the ``internalratelimiter`` is always on.

Reply:
We halen hem weg uit de config.